examples/events/test-cases.yaml

```
@@ -13,9 +13,10 @@ test1:
     # Parameters for 'Set Groupers' using task `set_groupers`.
     groupers:
       groupers: # (list[ecoscope_workflows.tasks.groupby._groupby.Grouper], FieldInfo(annotation=NoneType, required=True, description='            Index(es) and/or column(s) to group by, along with\n            optional display names and help text.\n            '))
-        - index_name: "month"
-          display_name: "Month"
-          directive: "%B"
+        - temporal_index:
+            directive: "%B"
+            selector_title: "Month (Example: September)"
+            display_name: "Month"
```

@atmorling I would expect this to reflect 1:1 what the RJSF input needs to be.

```suggestion
        - temporal_index:
            selector_title: "Month (example: September)"
```
?